Zilon.Core/Zilon.TextClient/Program.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.DependencyInjection;
+
+using Zilon.Core.Client;
+using Zilon.Core.Commands;
+using Zilon.Core.Players;
+using Zilon.Core.Tactics;
+using Zilon.Core.Tactics.Behaviour;
+using Zilon.Core.Tactics.Spatial;
+using Zilon.Core.World;
+
+namespace Zilon.TextClient
+{
+    static class Program
+    {
+        static async System.Threading.Tasks.Task Main()
+        {
+            var serviceContainer = new ServiceCollection();
+            var startUp = new StartUp();
+            startUp.RegisterServices(serviceContainer);
+
+            serviceContainer.AddSingleton<IGlobeInitializer, GlobeInitializer>();
+            serviceContainer.AddSingleton<IGlobeExpander>(provider => (BiomeInitializer)provider.GetRequiredService<IBiomeInitializer>());
+            serviceContainer.AddSingleton<IGlobeTransitionHandler, GlobeTransitionHandler>();
+            serviceContainer.AddSingleton<IPersonInitializer, HumanPersonInitializer>();
+            serviceContainer.AddSingleton<IPlayer, HumanPlayer>();
+            serviceContainer.AddScoped<MoveCommand>();
+
+            using var serviceProvider = serviceContainer.BuildServiceProvider();
+
+            // Create globe
+
+            using var scope = serviceProvider.CreateScope();
+
+            var globeInitializer = scope.ServiceProvider.GetRequiredService<IGlobeInitializer>();
+            var globe = await globeInitializer.CreateGlobeAsync("intro");
+            var player = scope.ServiceProvider.GetRequiredService<IPlayer>();
+
+            var gameLoop = new GameLoop(globe);
+            var uiState = scope.ServiceProvider.GetRequiredService<ISectorUiState>();
+            var playerActor = (from sectorNode in globe.SectorNodes
+                               from actor in sectorNode.Sector.ActorManager.Items
+                               where actor.Person == player.MainPerson
+                               select actor).SingleOrDefault();
+            var playerActorSectorNode = (from sectorNode in globe.SectorNodes
+                                         from actor in sectorNode.Sector.ActorManager.Items
+                                         where actor.Person == player.MainPerson
+                                         select sectorNode).SingleOrDefault();
+
+            // This is code smells. It is not good settings
+            player.BindSectorNode(playerActorSectorNode);
+
+            uiState.ActiveActor = new ActorViewModel { Actor = playerActor };
+
+            // Play
+
+            gameLoop.StartProcessAsync();
+
+            do
+            {
+                var inputText = Console.ReadLine();
+                if (inputText.StartsWith("m"))
+                {
+                    var taskSource = scope.ServiceProvider.GetRequiredService<IActorTaskSource<ISectorTaskSourceContext>>();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'taskSource'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)